### PR TITLE
VS Code Desktop docs update: connecting screenshots, reconnecting, and optimisations. 

### LINKED ISSUE #2262: Make VS Code Desktop preference option more prominent
### Describe your feature request

Not all users are discovering VS Code Desktop user preference, this is to update the docs to make a stronger callout, possibly with a GIF that shows how to setup the user preferences for VS Code Desktop. 

Also link to SSH where necessary: https://www.gitpod.io/docs/configure/ssh

### LINKED ISSUE #2461: Document VS Code Desktop popups (how to optimise)
Document: 
1. SSH fingerprint
2. Trust the workspace
3. Allowing the VS Code Desktop redirect
4. etc

Either explain how to optimise, or why it cannot be updated / changed. 

Relates to: 
- https://github.com/gitpod-io/gitpod/issues/9568
- https://github.com/gitpod-io/website/issues/2455

Internal link [[1](https://gitpod.slack.com/archives/C01KGM9BH54/p1659517419239619)]

### PR DESCRIPTION
**Pull request by loujaybee**
Update instructions on: 

1. Optimising the steps to open VS Code Desktop 
2. Document reconnection instructions for VS Code
3. Improve documentation about opening within VS Code vs via Splash page 

- [x] Document Workspace Trust modal
- [x] Document SSH fingerprint modal
- [x] Add alt names for images
- [ ] ~Document browser redirect prompt modal~ (https://github.com/gitpod-io/gitpod/pull/12082)

Closes #2262
Closes #2461

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2490"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2490"></a>

### CODE REVIEW

File: gitpod/docs/ides-and-editors/vscode.md [hunk 1]
```diff
@@ -128,6 +147,46 @@ The process of installing fonts matches how you typically install custom fonts l
    }
    ```
 
+## Optimising VS Code Desktop
+
+If you're using VS Code Desktop for frequent work you'll want to optimise your set up. Below are some tips to get a workspace set up as efficient as possible.
+
+<!-- TODO: Browser redirect popup -->
```
Comment (loujaybee) [inline gitpod/docs/ides-and-editors/vscode.md:154]: Still haven't figured this one out on Mac + Chrome 😬

File: gitpod/docs/ides-and-editors/vscode.md [hunk 2]
```diff
@@ -128,6 +147,46 @@ The process of installing fonts matches how you typically install custom fonts l
...
+<!-- TODO: Platform (Linux, etc) popup -->
```
Comment (loujaybee) [inline gitpod/docs/ides-and-editors/vscode.md:155]: I am no longer seeing this, is it suppressed now? Need to test more.

File: gitpod/docs/ides-and-editors/vscode.md [hunk 3]
```diff
@@ -128,6 +147,46 @@ The process of installing fonts matches how you typically install custom fonts l
...
+
+### SSH fingerprint
```
Comment (loujaybee) [inline gitpod/docs/ides-and-editors/vscode.md:157]: Wasn't sure if it's worth mentioning how to do this for other methods (e.g. how to literally update known hosts), or whether to reference just the latest method of SSH Gateway for VS Code Desktop. I went with the latter for now, but would be curious incase anyone thinks differently.

File: gitpod/docs/ides-and-editors/vscode.md [hunk 4]
```diff
@@ -128,6 +147,46 @@ The process of installing fonts matches how you typically install custom fonts l
...
+
+![Accepting the SSH fingerprint in VS Code](/images/editors/vscode-ssh-fingerprint.png)
+_Accepting the SSH fingerprint in VS Code_
+
+It is common convention to be asked for a fingerprint when accessing a new SSH server. When the fingerprint is accepted, a reference to the server is stored in your local `known_hosts` file, which suppresses subsequent SSH connection prompts for that server.
+
+Gitpod uses the SSH protocol
```
Comment (filiptronicek) [inline gitpod/docs/ides-and-editors/vscode.md:164]: suggestion: Delete this. Is it a leftover? 👀

```suggestion
```

File: gitpod/docs/ides-and-editors/vscode.md [hunk 5]
```diff
@@ -27,17 +27,22 @@ There are currently two supported ways to open a VS Code Desktop session connect
...
 3. Restart any running workspaces
 
-When the workspace starts, you will be prompted to open VS Code Desktop.
+When the workspace starts, you will be prompted to open VS Code Desktop. You can also access your workspace using VS Code Browser, or copy SSH credentials from this page.
 
-<!-- TODO: Add image -->
+![Open VS Code Desktop from the workspace start page](/images/editors/vscode-desktop-splash.png)
+_Open VS Code Desktop from the workspace start page_
+
+![Selecting VS Code Desktop as an editor preference](/images/editors/select-vscode-desktop-preference.png)
```
Comment (filiptronicek) [inline gitpod/docs/ides-and-editors/vscode.md:44]: suggestion (minor): Can we add the dark mode version of these? 

Not really sure yet about how to do it, so didn't add as a suggestion, sorry!

1.
| Light | Dark |
|-------|------|
| image psd(16) | image psd(15) |



2.
image

**Review by filiptronicek — COMMENTED** [review]
Took a look and didn't find any issues with the content itself, just some other suggestions. Well done! 👏

File: gitpod/docs/ides-and-editors/vscode.md (hunk 5 above)
Comment (loujaybee) [reply gitpod/docs/ides-and-editors/vscode.md]: Done!

**Review by akosyakov — APPROVED** [review]

**Review by filiptronicek — APPROVED** [review]
LGTM ✅

**Review by jeanp413 — APPROVED** [review]

**Comment by loujaybee** [comment]
Thanks @jeanp413, @akosyakov, @filiptronicek ! 🚀

**Pull request merged**